File: Code Chef/JavaScript/400-499/451 Broken Phone BROKENPHONE.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.split(" ").map(Number));
    
    return intArray;
};

const whatToDoRepairOrBuyNew = (phoneRepairCost, phoneBuyCost) => {
    (phoneRepairCost === phoneBuyCost)
        ? console.log("ANY")
        : (phoneRepairCost < phoneBuyCost)
            ? console.log("REPAIR")
            : console.log("NEW PHONE");
}

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n"));
    
    testcases.forEach((testcase) => {
        const [phoneRepairCost, phoneBuyCost] = testcase;
        whatToDoRepairOrBuyNew(phoneRepairCost, phoneBuyCost);
    });
});

File: Code Chef/C Programming/400-499/451 Broken Phone BROKENPHONE.c
#include <stdio.h>
int main() 
{
    int t,i,x,y;
    scanf("%d",&t);
    for(i=0;i<t;i++)
    {
        scanf("%d%d",&x,&y);
        if(x<y)
        {
            printf("REPAIR\n");
        }
        else if (y<x)
        {
            printf("NEW PHONE\n");
        }
        else
        {
            printf("ANY\n");
        }
    }
	return 0;
}